src/Services/GoogleAnalytics.js

```diff
@@ -0,0 +1,162 @@
+
+import ua from 'universal-analytics'
+
+const { app } = require('electron').remote
+
+const ganacheAnalyticsId = 'UA-83874933-5'
```

Comment:
Ideally this wouldn't be checked into our source repo, as otherwise people's local builds, etc, will report potentially muddled stats. Ideally any of the IDs/Keys necessary to init our analytics drivers should be in a private repo that's a git submodule.

Reply:
I agree, but I think we should address this another time